Src/Sarif.PatternMatcher/Fingerprint.cs

```diff
@@ -167,6 +171,11 @@ public override string ToString()
                 components.Add($"[{PersonalAccessTokenKeyName}={this.PersonalAccessToken.Trim()}]");
             }
 
+            if (!string.IsNullOrEmpty(Port))
```

Comment:
>Port [](start = 38, length = 4)

i wouldn't add this component. instead, I'd work this into the host value, which supports a port value. what do you think?

Reply:
for sqlserver that is true, but for the pr james is working (postgres) the port is another field and can be separate from the host.
Server=servername;User Id=username;Port=1234;Password=xyz.
For sqlserver, the host will contain server + port if exists

---
In reply to: [569697684](https://github.com/microsoft/sarif-pattern-matcher/pull/160#discussion_r569697684) [](ancestors = 569697684)

Reply:
My idea is that we consider using the URI authority thing for this. i.e., we could bundle the user and port information with host.

[userinfo@]host[:port]

---
In reply to: [569699026](https://github.com/microsoft/sarif-pattern-matcher/pull/160#discussion_r569699026) [](ancestors = 569699026,569697684)

Reply:
@jameswinkler we will have to take a look at this in ur pr.

---
In reply to: [569768638](https://github.com/microsoft/sarif-pattern-matcher/pull/160#discussion_r569768638) [](ancestors = 569768638,569699026,569697684)